File: c/grade.c
#include<stdio.h>
int main()
{
int grade, p, c, m, e, b, per;

printf("Enter the grades of 5 subjects (phys, chem, maths, eng, bio): ");
scanf("%d%d%d%d%d",&p,&c,&m,&e,&b);

per=((int)(p+c+m+e+b+4.9)/5)*100;

if(per>59)
  printf("Grade: First division\n");
else if(per>49)
  printf("Grade: Second division\n");
else if(per>39)
  printf("Grade: Third division\n");
else 
  printf("Grade: Fail\n");
  
return 0;
}

File: java/Grade.java
import java.util.Scanner;

public class Grade {
    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);
        
        System.out.println("Enter marks: ");
        int marks = s.nextInt();

        if(marks >= 90 && marks <= 100){
            System.out.println("Grade: O");
        }
        else if(marks >= 80 && marks < 90){
            System.out.println("Grade: E");
        }
        else if(marks >= 70 && marks < 80){
            System.out.println("Grade: A");
        }
        else if(marks >= 60 && marks < 70){
            System.out.println("Grade: B");
        }
        else if(marks >= 50 && marks < 60){
            System.out.println("Grade: C");
        }
        else if(marks < 50){
            System.out.println("Grade: Fail");
        }
        else{
            System.out.println("Invalid!");
        }

        s.close();
    }
}
